src/main/java/io/stargate/sgv2/jsonapi/service/resolver/DeleteOneCommandResolver.java

```diff
@@ -131,6 +133,23 @@ private FindCollectionOperation getFindOperation(
           false);
     }
 
+    // BM25 search / sort?
+    SortExpression bm25Expr = SortClauseUtil.resolveBM25Search(sortClause);
+    if (bm25Expr != null) {
+      throw ErrorCodeV1.INVALID_SORT_CLAUSE.toApiException(
+          "BM25 search is not yet supported for this command");
+      // Likely implementation of [data-api#1939] to support BM25 sort
```

Comment:
Adding these to help with #1939 -- mostly will just need testing. But don't want to delay merging of this PR any longer.